staff/marta-cervera/challenges/calculator-app/calculadora. 2/README.md

```diff
@@ -0,0 +1,16 @@
+# Calculator App
+
+It is a calculator that it can be used by everyone who wants to do an operation. 
+
+![Alt Calculator](./calculadora.png)
+
+## How it works
+The user who wants to use it must click on a number, after clik again on the operation that has to be done and finally the second number. The user must push the button equal to know the result of the operation. Also it has a button, for reset the calculator.
+## Technical Description 
+### What Operations can do?
+- Add
+- Substract
+- Divide
+- Multiply
+## Index.html
+The app structure is rectangle that inside, it has a structure of buttons taht corresponds with numbers and operations. Above the buttos there is a place in which area, the numbers are going to appear as soon as you press the botton.
```

Comment:
small typos: taht -> that / button -> buttons

Reply:
Thank you Maria Jesús. Resolved